src/routes/test-contentful-content/+page.server.ts

```diff
@@ -16,11 +17,22 @@ const query = gql`
   }
 `;
 
-export async function load(): Promise<Document> {
-  const data = await contentfulFetch<EntryQuery>(printQuery(query));
+export const load = async () => {
+  const client = getContentfulClient({
+    spaceID: CONTENTFUL_SPACE_ID,
+    token: CONTENTFUL_DELIVERY_API_TOKEN,
+  });
+  const data = await client.fetch<EntryQuery>(printQuery(query));
   if (data) {
-    return data?.testRichText?.body?.json;
+    const document = data?.testRichText?.body?.json as Document | undefined | null;
+    return {
+      query,
+      document: document || markdownDocument.document,
+    };
```

Comment:
What's the reasoning behind returning `query` here? I'm not seeing it used in the corresponding `+page.svelte`.

Reply:
This is an artifact of when the previewing was going to use the query to request the preview data on the front-end. I'll remove it.

Reply:
^ don't forget about this!